### PR TITLE
[WIP] Change config so all specs are screenshot to assist with e2e troubles…

### PR DESCRIPTION
**Pull request by rhamilto**
…hooting

### CODE REVIEW

File: frontend/integration-tests/protractor.conf.ts [hunk 1]
```diff
@@ -15,7 +15,7 @@ const tap = !!process.env.TAP;
 export const appHost = `${process.env.BRIDGE_BASE_ADDRESS || 'http://localhost:9000'}${(process.env.BRIDGE_BASE_PATH || '/').replace(/\/$/, '')}`;
 export const testName = `test-${Math.random().toString(36).replace(/[^a-z]+/g, '').substr(0, 5)}`;
 
-const htmlReporter = new HtmlScreenshotReporter({dest: './gui_test_screenshots', inlineImages: true, captureOnlyFailedSpecs: true, filename: 'test-gui-report.html'});
+const htmlReporter = new HtmlScreenshotReporter({dest: './gui_test_screenshots', inlineImages: true, filename: 'test-gui-report.html'});
```
Comment (alecmerdler) [inline frontend/integration-tests/protractor.conf.ts]: Will artifact storage size be an issue?

Comment (spadgett) [reply frontend/integration-tests/protractor.conf.ts]: We don't plan to enable this by default. It's only for debugging some of the failures we're seeing in CI.

**Pull request closed without merging**